server/schedule/operator/create_operator.go

```diff
@@ -95,7 +95,11 @@ func CreateMoveLeaderOperator(desc string, cluster opt.Cluster, region *core.Reg
 }
 
 // CreateSplitRegionOperator creates an operator that splits a region.
-func CreateSplitRegionOperator(desc string, region *core.RegionInfo, kind OpKind, policy pdpb.CheckPolicy, keys [][]byte) *Operator {
+func CreateSplitRegionOperator(desc string, region *core.RegionInfo, kind OpKind, policy pdpb.CheckPolicy, keys [][]byte) (*Operator, error) {
+	if core.IsInJointState(region.GetPeers()...) {
+		return nil, errors.Errorf("cannot split region which is in joint state")
```

Comment:
We need modify the error according to the new error standard.

Reply:
cc @HunDunDM

Reply:
I think the `errors` under `operator` should be uniformly modified in another PR.

Reply:
Does the other errors do not meet the error code standard code?

Reply:
Yes, so this part of the modification requires cherry-pick.